### PR TITLE
chore(plugin-server): roll out batching ingestion by id to 7964

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

Rolling out https://github.com/PostHog/posthog/pull/10071 to the test team.

Analysis on batch sizes tells me that we're almost always working off of batches of 1, except, after releases, when it goes up to small hundreds: https://metrics.posthog.com/d/CD1kWjLnz/plugin-server-new?orgId=1&viewPanel=50 but even then we'd get about double the batches if we split them by id, which is fine. 

batch total length refers to what we get in eachBatch https://github.com/PostHog/posthog/blob/master/plugin-server/src/main/ingestion-queues/batch-processing/each-batch.ts#L21

slack: https://posthog.slack.com/archives/C0374DA782U/p1654862274568649

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: task-definition.plugins-async.json [hunk 1]
```diff
@@ -115,6 +115,10 @@
                 {
                     "name": "WORKER_CONCURRENCY",
                     "value": "2"
+                },
+                {
+                    "name": "INGESTION_BATCH_BREAKUP_BY_DISTINCT_ID_TEAMS",
```
Comment (macobo) [inline task-definition.plugins-async.json:120]: We don't need this in this file.

Comment (tiina303) [reply task-definition.plugins-async.json:120]: assuming we don't do future trickery and run ingestion there 😅 - I wonder if we should have a plugin-server common config and then async and ingestion separate parts, which would make it easier to see stuff from quick looks

Comment (Twixes) [reply task-definition.plugins-async.json]: Given that this will only be in for a short time anyway, I removed this.

**Review by Twixes — APPROVED** [review]
Let's get this in and roll out fully soon

**Pull request merged**